torch/_functorch/_aot_autograd/schemas.py

```diff
@@ -636,6 +636,7 @@ class AOTConfig:
     aot_autograd_arg_pos_to_source: Optional[List[Source]] = None
     inference_compiler: Optional[Callable] = None
     enable_log: bool = True
+    pre_dispatch: bool = False
```

Comment:
add a comment here that this is guaranteed to be False when we're not doing export